.codespell/.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = .git,node_modules,yarn.lock,Cargo.lock,./typescript/helloworld
```

Comment:
There's some bug with codespell causing it to trip on `receivedFrom` in helloworld even though it's on the ignore list 🤷 
Other than that one occurrence it works fine.